toolkit/tools/imagegen/configuration/parse_partition.go

```diff
@@ -89,7 +89,17 @@ func processDisk(inputDiskValue string) (err error) {
 			disks = append(disks, Disk{})
 		}
 
-		disks[latestDiskIndex].PartitionTableType = PartitionTableTypeGpt
+		// Read from /proc/cmdline to determine if the user is intending for a GPT partition
+		partitionTableType, err := GetKernelCmdLineValue("--gpt", 6)
```

Comment:
Does `--gpt` actually have any value associated with it?  Or is it just a flag?

Reply:
Thanks for capturing this. There was a typo in the code. But "--gpt" does have value that comes with it, which is a boolean.